src/runner/runner.ts

```diff
@@ -50,7 +46,7 @@ const runRule = (
         rule,
         exceptRuleByLocations,
       );
-    } else {
+    } else if (isObject(target)) {
```

Comment:
a bugfix.
I realized that `given: $` would never be executed for primitive values, which is totally not intended.
An edge case, but still.